input/pagecontent/auereqdi.md

```diff
@@ -1,10 +1,338 @@
-### AU eRequesting Data for Interoperability in AU eRequesting R1
+[Australian eRequesting Data for Interoperability (AUeReqDI)](https://sparked.csiro.au/index.php/sparked-products-resources/auereqdi/auereqdi-release-1/) is the product of a national clinician focussed requirements gathering project operating as part of the [Sparked AU FHIR Accelerator](https://sparked.csiro.au/).  AUeReqDI outputs form a set of data requirements to be considered and referred to as part of the development and definition of AU eRequesting. 
 
-TBD
+The primary intent of AUeReqDI is to design and govern a collection of coherent, reusable building blocks known as ‘data groups’. These data groups specify “what” the clinical requirements of the clinical information that should be included for eRequests. However, it does not specify “how” the data is exchanged; this is the role fulfilled by the FHIR standard. AUeReqDI is not required to be implemented as a whole single product.  AUeReqDI builds upon and complements the foundational [Australian Core Data for Interoperability (AUCDI)](https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/aucdi-release-1/) and focuses on the specific use case of eRequesting.
 
-The table below shows the relationship between the AUeReqDI Data Groups and Elements and AU eRequesting profiles.<br/>
+AUeReqDI Release 1 (R1) is focused on an agreement of the minimum data required to support standardised eRequesting within the Australian health context, and forms a common language foundation that allows systems to exchange semantically accurate data for eRequests. It incorporates and builds upon prior work from national and international programs and initiatives, including the Royal College of Pathologists of Australiasia (RCPA)'s Pathology information, Terminology and Units Standardisation (PITUS) framework and the Royal Australian New Zealand College of Radiology (RANZCR's) Radiology Referral Set.
+
+With AUeReqDI defining clinical data requirements and FHIR AU eRequesting providing the Implementation Guide for FHIR-based electronic requesting of diagnostic imaging and pathology services, an interpretation of AUeReqDI is necessary which is undertaken through the community.
+
+Updates to AU eRequesting depend upon community input through the evolution of AUCDI and AU Core. We encourage requesting any necessary clarifications to AUeReqDI or AUCDI through through the respective process that helps inform future updates to FHIR AU eRequesting and AU Core. In addition, we encourage our audience to submit questions and feedback to AU eRequesting or AU Core specifications by clicking on the Propose a change link in the footer of every page.
```

Comment:
I see the inclusion of AUCDI & AU Core but it's giving the impression that changes to eRequesting depend solely on those two.

Suggest the first sentence is instead:
"Updates to AU eRequesting and AU Core depend upon community input and we encourage our audience to submit questions and feedback on implementation guides by clicking on the Propose a change link in the footer of every page."

If the first sentence is adjusted as above then remove the last sentence in the paragraph.

Reply:
Amended as suggested.

Reply:
Need to push the commit with this change.

Reply:
Sorry for the delay, having an issue with my build locally and was trying to fix. No issue on build.fhir: https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/ft_ereqdi-draft/auereqdi.html.